servlet/src/main/scala/org/http4s/servlet/Http4sServlet.scala

```diff
@@ -42,8 +43,17 @@ abstract class Http4sServlet[F[_]](
     service: HttpApp[F],
     servletIo: ServletIo[F],
     dispatcher: Dispatcher[F],
-)(implicit F: Async[F])
+)(implicit F: Sync[F])
     extends HttpServlet {
+
+  @deprecated("Binary compatibility", "0.23.12")
+  private[servlet] def this(
```

Comment:
@armanbilge, do I recall that you discovered a trick for the static forwarder on these?  This is the one that's going to fail.

Reply:
This is a constructor, so it shouldn't have a static forwarder right?

Reply:
Shit. I linked the wrong one.  I meant [this one](https://github.com/http4s/http4s-servlet/pull/24/files#diff-4e297c4945c6c3b66ce3d74472f332e6d0f5a79a56ee483c2a3f6dafc57f0c2aR104).

Reply:
That link isn't being very useful.  Line 104 of `BlockingHttp4sServlet`.